keyboards/tkw/stoutgat/v2/keymaps/qmk/keymap.json

```diff
@@ -0,0 +1,158 @@
+{
```

Comment:
If you're intending for this to be your personal keymap, the folder should be `keyboards/tkw/stoutgat/v2/keymaps/thys/`.

If you're intending for it to be the default for QMK Configurator:

1. The file to achieve that doesn't go in this repository.
2. It would need to be updated due to the change to `LAYOUT_encoder`.
3. It doesn't match the assignment used in the `default` keymap, which is preferred.

Reply:
removed